charts.d/apache.chart.sh

```diff
@@ -98,16 +98,17 @@ apache_detect() {
 
 apache_get() {
 	local oIFS="${IFS}" ret
+	# shellcheck disable=2207
 	IFS=$':\n' apache_response=($(run curl -Ss ${apache_curl_opts} "${apache_url}"))
 	ret=$?
 	IFS="${oIFS}"
 
-	[ $ret -ne 0 -o "${#apache_response[@]}" -eq 0 ] && return 1
```

Comment:
Why is this needed? Readability is killed with this.

Reply:
I would argue that having multiple operators starting with `-` kills readability. Using `[ ]` neatly groups conditions and `&&` or `||` makes it clear what is a condition and how do they apply.

And as with every shellcheck code, there is an explanation: https://github.com/koalaman/shellcheck/wiki/SC2166

Reply:
ok